src/Slingshot.vala

```diff
@@ -19,6 +19,8 @@
 public class Slingshot.Slingshot : Wingpanel.Indicator {
     private SlingshotView? view = null;
 
+    private Gtk.Grid? indicator_grid = null;
+    private Gtk.Image? indicator_icon = null;
```

Comment:
Probably only need to have `indicator_grid` up here. The reason that it's done like this is because it's a "singleton" which means that we only want to ever create it once and once we've created it we always re-use the same one. That's what the null check on line 68 does. If we know that we only create `indicator_grid` once, then we know that we only create what's inside it once as well (hence no null checks) and so instead of making nullable ( that's what ? means) private variables we can just use the `var` keyword like `var indicator_label = new Gtk.Label ()`